npm_integration_frameworks/reactnative/android/gradle.properties

```
@@ -0,0 +1,5 @@
+ActivityStarter_kotlinVersion=1.7.0
+ActivityStarter_minSdkVersion=21
+ActivityStarter_targetSdkVersion=31
+ActivityStarter_compileSdkVersion=31
```

On va preferer reprendre les versions de la lib core plutot que de l'appli de demo (qui est plus elevee uniquement pour pouvoir passer sur le play store):

```suggestion
ActivityStarter_minSdkVersion=19
ActivityStarter_targetSdkVersion=28
ActivityStarter_compileSdkVersion=28
```